Macros.h

```diff
@@ -1,7 +1,6 @@
 #ifndef HEADER_Macros
 #define HEADER_Macros
 
-#include <assert.h> // IWYU pragma: keep
```

Comment:
Please keep this as using `HTOP_ASSERT` requires this header to be included. Removing this line scatters this all over the place unnecessarily.

Reply:
dropped